docs/content/hunting.md

```diff
@@ -1,76 +1,240 @@
+
+   ```sh
+   ord wallet identify
+   ```
+
+### Searching for Rare Ordinals in a Wallet that Exports Multi-path Descriptors
+
+Some descriptors describe multiple paths in one descriptor using angle brackets,
+e.g., `<0;1>`. Multi-path descriptors are not yet supported by Bitcoin Core, so
+you'll first need to convert them into multiple descriptors, and then import
+those multiple descriptors into Bitcoin Core.
+
+1. First get the multi-path descriptor from your wallet. It will look something
+   like this:
+
+   ```
+   wpkh([bf1dd55e/84h/0h/0h]xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/<0;1>/*)#fw76ulgt
+   ```
+
+2. Create a descriptor for the receive address path:
+
+   ```
+   wpkh([bf1dd55e/84'/0'/0']xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/0/*)
+   ```
+
+   And the change address path:
+
+   ```
+   wpkh([bf1dd55e/84'/0'/0']xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/1/*)
+   ```
+
+3. Get and note the checksum for the receive address descriptor, in this case
+   `tpnxnxax`:
+
+   ```sh
+   bitcoin-cli getdescriptorinfo \
+     'wpkh([bf1dd55e/84h/0h/0h]xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/0/*)'
+   ```
+
+   ```json
+   {
+     "descriptor": "wpkh([bf1dd55e/84'/0'/0']xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/0/*)#csvefu29",
+     "checksum": "tpnxnxax",
+     "isrange": true,
+     "issolvable": true,
+     "hasprivatekeys": false
+   }
+   ```
+
+   And for the change address descriptor, in this case `64k8wnd7`:
+
+   ```sh
+   bitcoin-cli getdescriptorinfo \
+     'wpkh([bf1dd55e/84h/0h/0h]xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/1/*)'
+   ```
+
+   ```json
+   {
+     "descriptor": "wpkh([bf1dd55e/84'/0'/0']xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/1/*)#fyfc5f6a",
+     "checksum": "64k8wnd7",
+     "isrange": true,
+     "issolvable": true,
+     "hasprivatekeys": false
+   }
+   ```
+
+4. Now import the descriptors, with the correct checksums, into Bitcoin Core.
+
+   First the receive address descriptor:
+
+   ```sh
+   bitcoin-cli \
+    importdescriptors \
+    '[{ "desc": "wpkh([bf1dd55e/84h/0h/0h]xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/0/*)#tpnxnxax", "timestamp":0}]'
+   ```
+
+   And then the change address descriptor:
+
+   ```sh
+   bitcoin-cli \
+    importdescriptors \
+    '[{ "desc": "wpkh([bf1dd55e/84h/0h/0h]xpub6CcJtWcvFQaMo39ANFi1MyXkEXM8T8ZhnxMtSjQAdPmVSTHYnc8Hwoc11VpuP8cb8JUTboZB5A7YYGDonYySij4XTawL6iNZvmZwdnSEEep/1/*)#64k8wnd7", "timestamp":0}]'
```

Comment:
I think you import two descriptors at once. It's probably faster like this since this command could take a while